src/backend/main.ts

```diff
@@ -490,6 +492,71 @@ addOneTimeListener('frontendReady', () => {
     }
   }
 
+  if (isNixPkg) {
+    const nixPkgWarning: Electron.MessageBoxOptions = {
+      title: i18next.t(
+        'box.warning.nixpkg.title',
+        'Heroic is running from community nix package'
+      ),
+      message: i18next.t('box.warning.nixpkg.message', {
+        defaultValue:
+          'You may experience issues not present with the official package.{{newLine}}If you experience issues, please first check that it also happens with one of the official packages before requesting support from the Heroic Discord.{{newLine}}If your issue only occurs with the Nix Heroic package, raise an issue on the nixpkgs GitHub repository.',
+        newLine: '\n'
+      }),
+      checkboxLabel: i18next.t('box.warning.nixpkg.checkbox', {
+        defaultValue: 'Do not show this message again'
+      }),
+      checkboxChecked: false
+    }
+
+    const showNixPkgWarning = configStore.get('showNixPkgWarning', true)
+
+    if (showNixPkgWarning) {
+      dialog
+        .showMessageBox({
+          ...nixPkgWarning
+        })
+        .then((result) => {
+          if (result.checkboxChecked) {
+            configStore.set('showNixPkgWarning', false)
+          }
+        })
+    }
+  }
+
+  usingNixUmu().then((result) => {
+    if (result) {
+      const nixUmuWarning: Electron.MessageBoxOptions = {
+        title: i18next.t(
+          'box.warning.nixumu.title',
+          'UMU running from the nix package'
+        ),
+        message: i18next.t('box.warning.nixumu.message', {
+          defaultValue:
+            'If you experience issues launching games using UMU, please first check that it also happens with UMU managed by Heroic, after removing UMU from your system environment.'
```

Comment:
"umu" is not an acronym (we're probably making this mistake somewhere else in the codebase still, but no need to repeat it)
```suggestion
            'If you experience issues launching games using umu, please first check that it also happens with umu managed by Heroic, after removing umu from your system environment.'
```
(the title also needs changing)

Reply:
Fixed.  (It is all caps in the UI, which is why I made this mistake.)